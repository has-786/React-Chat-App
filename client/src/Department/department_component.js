import SideNavBarDept from "./sidenavbar_dept";
import PdfDisplayDept from "./pdfdisplay_dept";
import "./css/main.css";
import React from 'react';
export default class Department extends React.Component{

  constructor(props){
    super(props);
    this.state={
      pdfs:[{id:'ana',link:'http://www.bjmcpune.org/ANATOMY.pdf'},{id:'physio',link:'http://www.bjmcpune.org/ANATOMY.pdf'}],
      link:""
    }
    this.changePDF=this.changePDF.bind(this);
  }

  render(){
    return <div class="conatiner-fluid">
    <div class="row">
    <div class="col-lg-4 sidenavbar" id="sidenav"><SideNavBarDept changePDF={this.changePDF}/></div>
    <div class="col-lg-8 pdfdisplay" id="pdfdisplay"><PdfDisplayDept link={this.state.link}/></div>
    </div>
    </div>;
}

 changePDF(id)
{
//  alert(id);
    this.setState({link:this.state.pdfs.find(p=>p.id==id).link});
}

}
