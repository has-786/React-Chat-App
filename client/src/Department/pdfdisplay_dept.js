import "./css/pdfdisplaydept.css";
export default function PdfDisplayDept(props)
{
    return <iframe src={props.link} id="pdf"></iframe>;
}
