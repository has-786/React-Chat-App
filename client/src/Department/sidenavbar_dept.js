import "./css/sidenavbardept.css"
import {Level1_expansion,Level2_expansion} from "./Javascript/sidenavbar_javascript";

/*export function Level1_expansion(param)
{
    document.getElementById(param).style.color="green";
}*/
export default function SideNavBarDept(props)
{

    return <nav class="navbar bg-dark navbar-dark">
        <ul class="navbar-nav navbar-brand " >
            {/*Department1 */}
            <li class="nav-item " onClick={e=>Level1_expansion('preClinicalSubitem')} >&gt; Pre-Clinical Departments</li>
            <ul class="navbar-nav collapsible level1 bg-success " id="preClinicalSubitem" >
                <li class="nav-item" id='ana' onClick={()=>props.changePDF('ana')}>Anatomy</li>
                <li class="nav-item">Physiology</li>
                <li class="nav-item">Biochemistry</li>
            </ul>
            {/*Department2 */}
            <li class="nav-item"  onClick={e=>Level1_expansion('paraClinicalSubitem')} >&gt; Para-Clinical Departments</li>
            <ul class="navbar-nav collapsible level1 bg-success" id="paraClinicalSubitem">
                <li class="nav-item">Microbiology</li>
                <li class="nav-item">Forensic Medicine</li>
                <li class="nav-item">Pharmacology</li>
                <li class="nav-item" >Community Medicine</li>
                <li class="nav-item">Pathology</li>
            </ul>
            {/*Department3 */}
            <li class="nav-item" onClick={e=>Level1_expansion('clinicalSubitem')}>&gt; Clinical Departments</li>
            <ul class="navbar-nav collapsible level1 bg-success" id="clinicalSubitem">
                <li class="nav-item" onClick={e=>Level2_expansion('medicalSubitem')}>&gt; Medical Specialties</li>
                <ul class="navbar-nav collapsible level2" id="medicalSubitem">
                    <li class="nav-item">General Medicine</li>
                    <li class="nav-item">Pediatrics</li>
                    <li class="nav-item">Tuberculosis and Chest Diseases</li>
                    <li class="nav-item">Skin & V. D.</li>
                    <li class="nav-item">Psychiatry</li>
                    <li class="nav-item">Radiology</li>
                    <li class="nav-item" >Anesthesiology</li>
                </ul>

                <li class="nav-item" onClick={e=>Level2_expansion('medicalSuperSubitem')}>&gt; Medical Super Specialties</li>
                <ul class="navbar-nav collapsible level2" id="medicalSuperSubitem">
                    <li class="nav-item">Cardiology</li>
                    <li class="nav-item">Neurology</li>
                    <li class="nav-item">Nephrology</li>
                </ul>

                <li class="nav-item" onClick={e=>Level2_expansion('surgicalSubitem')}>&gt; Surgical Specialties</li>
                <ul class="navbar-nav collapsible level2" id="surgicalSubitem">
                    <li class="nav-item"> General Surgery</li>
                    <li class="nav-item">Orthopedics</li>
                    <li class="nav-item">Ophthalmology</li>
                    <li class="nav-item">Otorhinolaryngology(ENT)</li>
                    <li class="nav-item">Obstetrics & Gynecology</li>
                </ul>

                <li class="nav-item" onClick={e=>Level2_expansion('surgicalSuperSubitem')}>&gt; Surgical Super Specialties</li>
                <ul class="navbar-nav collapsible level2" id="surgicalSuperSubitem">
                    <li class="nav-item"> Cardiovascular and Thoracic Surgery</li>
                    <li class="nav-item">Neuro-Surgery</li>
                    <li class="nav-item">Pediatric Surgery</li>
                    <li class="nav-item">Plastic Surgery</li>
                    <li class="nav-item">Urosurgery</li>
                </ul>

                <li class="nav-item" onClick={e=>Level2_expansion('alliedSubitem')}>&gt; Allied Services</li>
                <ul class="navbar-nav collapsible level2" id="alliedSubitem">
                    <li class="nav-item">Dentistry</li>
                    <li class="nav-item">Physiotherapy</li>
                    <li class="nav-item">Blood Bank</li>
                    <li class="nav-item">Eye Bank</li>
                    <li class="nav-item">Bio Medical Waste Management</li>
                    <li class="nav-item">Disaster Management</li>
                    <li class="nav-item">Nursing</li>
                    <li class="nav-item">Body / Kidney / Organ donation</li>
                    <li class="nav-item">Institute of Nursing education</li>
                </ul>

                <li class="nav-item" onClick={e=>Level2_expansion('hivSubitem')}>&gt; HIV-AIDS</li>
                <ul class="navbar-nav collapsible level2" id="hivSubitem">
                    <li class="nav-item">ICTC</li>
                    <li class="nav-item">ART</li>
                </ul>

            </ul>
        </ul>
    </nav>

}
