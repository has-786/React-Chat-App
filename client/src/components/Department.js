import React from 'react';
import Navbar from './navbar';
import Header from './header';
import "../css/New_Department.css"
import {Level1_expansion,Level2_expansion,menu_expansion} from './Department_javascript'
import PdfView from "./pdfView";
import image from "../right_arrow_small.png";
import url from "./url";

export default class Department extends React.Component {
    constructor(props){
        super(props);
        this.state={
            display:"",info:[

  {"name":"anatomy","link":"http://www.bjmcpune.org/ANATOMY.pdf"},
  {"name":"physiology","link":"http://www.bjmcpune.org/PHYSIOLOGY.pdf"},
  {"name":"biochemistry","link":"http://www.bjmcpune.org/BIOCHEMISTRY.pdf"},
  {"name":"microbiology","link":"http://www.bjmcpune.org/MICROBIOLOGY.pdf"},
  {"name":"forensicMedicine","link":"http://www.bjmcpune.org/FORENSIC%20MEDICINE.pdf"},
  {"name":"pharmacology","link":"http://www.bjmcpune.org/Website_Pharmacology_BJGMC%20Pune%202016_update_04-08-2016.pdf"},
  {"name":"communityMedicine","link":"http://www.bjmcpune.org/DEPT%20OF%20COMMUNITY%20MEDICINE.pdf"},
  {"name":"pathology","link":"http://www.bjmcpune.org/PATHOLOGY.pdf"},
  {"name":"generalMedicine","link":"http://www.bjmcpune.org/medicine.pdf"},
  {"name":"pediatrics","link":"http://www.bjmcpune.org/PEADIARICS.pdf"},
  {"name":"tuberculosis","link":"http://www.bjmcpune.org/CHEST%20TB.pdf"},
  {"name":"skin","link":"http://www.bjmcpune.org/SKIN.pdf"},
  {"name":"psychiatry","link":"http://www.bjmcpune.org/PSYCHIATRY.pdf"},
  {"name":"radiology","link":"http://www.bjmcpune.org/RADIODIAGNOSIS.pdf"},
  {"name":"anesthesiology","link":"http://www.bjmcpune.org/ANAESTHESIA.pdf"},
  {"name":"generalSurgery","link":"http://www.bjmcpune.org/SURGERY.pdf"},
  {"name":"orthopedics","link":"http://www.bjmcpune.org/ORTHOPEDICS.pdf"},
  {"name":"ophthalmology","link":"http://www.bjmcpune.org/Ophthalmology.pdf"},
  {"name":"otorhinolaryngology","link":"http://www.bjmcpune.org/ENT.pdf"},
  {"name":"obstetrics","link":"http://www.bjmcpune.org/OBSTRETICS%20AND%20GYENACOLOGY.pdf"},
  {"name":"cardiovascular","link":"http://www.bjmcpune.org/Department%20of%20CVTS.pdf"},
  {"name":"neuroSurgery","link":"http://www.bjmcpune.org/NEUROSURGERY.pdf"},
  {"name":"pediatricSurgery","link":"http://www.bjmcpune.org/PEDIATRIC%20SURGERY.pdf"},
  {"name":"plasticSurgery","link":"http://www.bjmcpune.org/PLASTIC%20SURGERY.pdf"},
  {"name":"urosurgery","link":"http://www.bjmcpune.org/Urology.pdf"},
  {"name":"dentistry","link":"http://www.bjmcpune.org/DENTAL.pdf"},
  {"name":"bloodBank","link":"http://www.bjmcpune.org/BLOOD%20BANK.pdf"},
  {"name":"eyeBank","link":"http://www.bjmcpune.org/EYE%20BANK.pdf"},
  {"name":"bioWaste","link":"http://www.bjmcpune.org/biomedical.pdf"}
            ]
    }
}

handleClick=name=>()=>{
    const elementsIndex = this.state.info.findIndex(element => element.name == name )

    this.setState({display:this.state.info[elementsIndex].link});
    menu_expansion('sidenav_list');

};

componentDidMount(){
    fetch(url+'/api/pdflinks',{ method:'GET'})
    .then(response=>{ return response.json()})
    .then((body)=>{
        this.setState({info:body});
     })
    .catch(err=>{});
}

    render(){
        return (<div id='mainBody'>
          <Header />
          <Navbar/>
        <section class="New_Department row">


            <nav class="side_navigation col-lg-3 col-md-3 ">

            <i style={{fontSize:'30px'}} class='fa fa-bars menu_collapse' onClick={(evt)=>menu_expansion('sidenav_list')}></i>
                <ul class="list-group sidenav_list"  >

                    {/*Department 1 Level1 */}
                    <li><a class="list-group-item list-group-item-action level1" data-toggle="list" onClick={e=>Level1_expansion('preClinicalSubitem')} ><img src={image}></img>Pre-Clinical Departments</a></li>
                    {/*Level 2 */}
                    <li><a class="list-group-item list-group-item-action level2 preClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('anatomy')}>Anatomy</a></li>
                    <li><a class="list-group-item list-group-item-action level2 preClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('physiology')}>Physiology</a></li>
                    <li><a class="list-group-item list-group-item-action level2 preClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('biochemistry')}>Biochemistry</a></li>

                    {/*Department 2 Level1 */}
                    <li><a class="list-group-item list-group-item-action level1" data-toggle="list"  onClick={e=>Level1_expansion('paraClinicalSubitem')}><img src={image}></img>Para-Clinical Department</a></li>
                    {/*Level 2 */}
                    <li><a class="list-group-item list-group-item-action level2 paraClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('microbiology')}>Microbiology</a></li>
                    <li><a class="list-group-item list-group-item-action level2 paraClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('forensicMedicine')}>Forensic Medicine</a></li>
                    <li><a class="list-group-item list-group-item-action level2 paraClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('pharmacology')}>Pharmacology</a></li>
                    <li><a class="list-group-item list-group-item-action level2 paraClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('communityMedicine')}>Community Medicine</a></li>
                    <li><a class="list-group-item list-group-item-action level2 paraClinicalSubitem" data-toggle="list" href="#Info1" onClick={this.handleClick('pathology')}>Pathology</a></li>


                    {/*Department 3 Level1 */}
                    <li><a class="list-group-item list-group-item-action level1" data-toggle="list"  onClick={e=>Level1_expansion('clinicalSubitem')}><img src={image}></img>Clinical Departments</a></li>
                    {/*Level 2 subitem 1*/}
                    <li><a class="list-group-item list-group-item-action level2 clinicalSubitem" data-toggle="list"  onClick={e=>Level2_expansion('medicalSpecialities')}><img src={image}></img>Medical Specialties</a></li>
                    {/*Level 3*/}
                    <li><a class="list-group-item list-group-item-action level3 medicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('generalMedicine')}>General Medicine</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('pediatrics')}>Pediatrics</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('tuberculosis')}>Tuberculosis and Chest Diseases</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('skin')}>Skin & V. D.</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('psychiatry')}>Psychiatry</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('radiology')}>Radiology</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('anesthesiology')}>Anesthesiology</a></li>

                    {/*Level 2 subitem 2*/}
                    <li><a class="list-group-item list-group-item-action level2 clinicalSubitem" data-toggle="list"  onClick={e=>Level2_expansion('medicalSuperSpecialities')}><img src={image}></img>Medical Super Specialties</a></li>
                     {/*Level 3*/}
                    <li><a class="list-group-item list-group-item-action level3 medicalSuperSpecialities" data-toggle="list" href="#ErrorInfo">Cardiology</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSuperSpecialities" data-toggle="list" href="#ErrorInfo">Neurology</a></li>
                    <li><a class="list-group-item list-group-item-action level3 medicalSuperSpecialities" data-toggle="list" href="#ErrorInfo">Nephrology</a></li>

                    {/*Level 2 subitem 3*/}
                    <li><a class="list-group-item list-group-item-action level2 clinicalSubitem" data-toggle="list"  onClick={e=>Level2_expansion('surgicalSuperSpecialities')}><img src={image}></img>Surgical Super Specialties</a></li>
                    {/*Level 3*/}
                    <li><a class="list-group-item list-group-item-action level3 surgicalSuperSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('cardiovascular')}>Cardiovascular and Thoracic Surgery</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSuperSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('neuroSurgery')}>Neuro-Surgery</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSuperSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('pediatricSurgery')}>Pediatric Surgery</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSuperSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('plasticSurgery')}>Plastic Surgery</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSuperSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('urosurgery')}>Urosurgery</a></li>

                    {/*Level 2 subitem 4*/}
                    <li><a class="list-group-item list-group-item-action level2 clinicalSubitem" data-toggle="list"  onClick={e=>Level2_expansion('surgicalSpecialities')}><img src={image}></img>Surgical Specialties</a></li>
                    {/*Level 3*/}
                    <li><a class="list-group-item list-group-item-action level3 surgicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('generalSurgery')}>General Surgery</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('orthopedics')}>Orthopedics</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('ophthalmology')}>Ophthalmology</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('otorhinolaryngology')}>Otorhinolaryngology</a></li>
                    <li><a class="list-group-item list-group-item-action level3 surgicalSpecialities" data-toggle="list" href="#Info1" onClick={this.handleClick('obstetrics')}>Obstetrics & Gynecology</a></li>

                    {/*Level 2 subitem 5*/}
                    <li><a class="list-group-item list-group-item-action level2 clinicalSubitem" data-toggle="list"  onClick={e=>Level2_expansion('alliedServices')}><img src={image}></img>Allied Services</a></li>
                    {/*Level 3*/}
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#Info1" onClick={this.handleClick('dentistry')}>Dentistry</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#ErrorInfo">Physiotherapy</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#Info1" onClick={this.handleClick('bloodBank')}>Blood Bank</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#Info1" onClick={this.handleClick('eyeBank')}>Eye Bank</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#Info1" onClick={this.handleClick('bioWaste')}>Bio Medical Waste Management</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#ErrorInfo">Disaster Management</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#ErrorInfo" >Nursing</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#ErrorInfo" >Body / Kidney / Organ donation</a></li>
                    <li><a class="list-group-item list-group-item-action level3 alliedServices" data-toggle="list" href="#Info1" onClick={this.handleClick('http://www.bjmcpune.org/OBSTRETICS%20AND%20GYENACOLOGY.pdf')}>Institute of Nursing education</a></li>

                    {/*Level 2 subitem 6*/}
                    <li><a class="list-group-item list-group-item-action level2 clinicalSubitem" data-toggle="list"  onClick={e=>Level2_expansion('hivAids')}><img src={image}></img>HIV AIDS</a></li>
                    {/*Level 3*/}
                    <li><a class="list-group-item list-group-item-action level3 hivAids" data-toggle="list" href="#ErrorInfo">ICTC</a></li>
                    <li><a class="list-group-item list-group-item-action level3 hivAids" data-toggle="list" href="#ErrorInfo">ART</a></li>


                </ul>
            </nav>

          <article class="col-lg-8 col-md-8">
            <header id="article_heading"> Departments</header>
            <br></br>
            
            <div class="tab-content">
            <div class="tab-pane fade col-lg-9 " id="Info1" >
                <PdfView link={this.state.display}/>
            </div>
            <div class="tab-pane fade col-lg-9 " id="ErrorInfo" >
               PDF NOT AVAILABLE !!!
            </div>
            </div>
        </article>

        </section>



        </div>)
    }
}
