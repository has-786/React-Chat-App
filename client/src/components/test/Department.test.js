import React from "react"
import Department from "../Department.js"
import renderer from 'react-test-renderer';
import {shallow,configure,mount} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({adapter:new Adapter()});

//Test 1
jest.mock('../Header',()=>'header')
jest.mock('../Navbar',()=>'navbar')
it('Snapshot test to check if renders correctly', () => {
  const component = renderer.create(<Department/>).toJSON();
  expect(component).toMatchSnapshot();
});

//Test 2
describe("Dept comp",() =>{
    test("Checking count of pdfs....",()=>{
        const wrapper= shallow(<Department/>);
       
      
       console.log("value of disp"+wrapper.state().info.name);
       
        expect(wrapper.state().info).toHaveLength(29);
    });
});


//Test 3
describe("Dept comp",() =>{
    test("Verify content of state....",()=>{
        const wrapper= shallow(<Department/>);
       wrapper.instance().handleClick('anatomy');
        expect(wrapper.state().info).toEqual(expect.arrayContaining([expect.objectContaining({name:"anatomy",link:"http://www.bjmcpune.org/ANATOMY.pdf"})]))    
        
    });
});
