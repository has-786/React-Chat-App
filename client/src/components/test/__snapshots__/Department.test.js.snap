// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test to check if renders correctly 1`] = `
<div
  id="mainBody"
>
  <header />
  <navbar />
  <section
    class="New_Department row"
  >
    <nav
      class="side_navigation col-lg-3 col-md-3 "
    >
      <i
        class="fa fa-bars menu_collapse"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "30px",
          }
        }
      />
      <ul
        class="list-group sidenav_list"
      >
        <li>
          <a
            class="list-group-item list-group-item-action level1"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Pre-Clinical Departments
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 preClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Anatomy
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 preClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Physiology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 preClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Biochemistry
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level1"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Para-Clinical Department
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 paraClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Microbiology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 paraClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Forensic Medicine
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 paraClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Pharmacology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 paraClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Community Medicine
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 paraClinicalSubitem"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Pathology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level1"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Clinical Departments
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 clinicalSubitem"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Medical Specialties
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            General Medicine
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Pediatrics
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Tuberculosis and Chest Diseases
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Skin & V. D.
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Psychiatry
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Radiology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Anesthesiology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 clinicalSubitem"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Medical Super Specialties
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSuperSpecialities"
            data-toggle="list"
            href="#ErrorInfo"
          >
            Cardiology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSuperSpecialities"
            data-toggle="list"
            href="#ErrorInfo"
          >
            Neurology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 medicalSuperSpecialities"
            data-toggle="list"
            href="#ErrorInfo"
          >
            Nephrology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 clinicalSubitem"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Surgical Super Specialties
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSuperSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Cardiovascular and Thoracic Surgery
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSuperSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Neuro-Surgery
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSuperSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Pediatric Surgery
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSuperSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Plastic Surgery
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSuperSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Urosurgery
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 clinicalSubitem"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Surgical Specialties
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            General Surgery
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Orthopedics
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Ophthalmology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Otorhinolaryngology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 surgicalSpecialities"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Obstetrics & Gynecology
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 clinicalSubitem"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            Allied Services
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Dentistry
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#ErrorInfo"
          >
            Physiotherapy
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Blood Bank
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Eye Bank
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Bio Medical Waste Management
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#ErrorInfo"
          >
            Disaster Management
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#ErrorInfo"
          >
            Nursing
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#ErrorInfo"
          >
            Body / Kidney / Organ donation
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 alliedServices"
            data-toggle="list"
            href="#Info1"
            onClick={[Function]}
          >
            Institute of Nursing education
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level2 clinicalSubitem"
            data-toggle="list"
            onClick={[Function]}
          >
            <img
              src="right_arrow_small.png"
            />
            HIV AIDS
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 hivAids"
            data-toggle="list"
            href="#ErrorInfo"
          >
            ICTC
          </a>
        </li>
        <li>
          <a
            class="list-group-item list-group-item-action level3 hivAids"
            data-toggle="list"
            href="#ErrorInfo"
          >
            ART
          </a>
        </li>
      </ul>
    </nav>
    <article
      class="col-lg-8 col-md-8"
    >
      <header
        id="article_heading"
      >
         Departments
      </header>
      <br />
      <div
        class="tab-content"
      >
        <div
          class="tab-pane fade col-lg-9 "
          id="Info1"
        >
          <iframe
            height="500px"
            id="pdf"
            src=""
            width="150%"
          />
        </div>
        <div
          class="tab-pane fade col-lg-9 "
          id="ErrorInfo"
        >
          PDF NOT AVAILABLE !!!
        </div>
      </div>
    </article>
  </section>
</div>
`;
